app/proguard-rules/okhttp3-rules.pro

```diff
@@ -0,0 +1,6 @@
+# OkHttp3
```

Comment:
Do you have a source for these rules, or did you derive them? If there's an external source, it's good to add a comment to the file to show where they come from. There's often many varied opinions on the required entries in Proguard rules for each library with which each author saying "This worked for me", so having traceability helps to know about the authority of the source and where to go if/when the library is updated.